polyfills/console/exception/polyfill.js

```diff
@@ -1 +1 @@
-console.exception = function exception() {};
+console.exception = 'error' in console ? console.error : function exception() {};
```

Comment:
I believe rhe function name would be different with this change, it'd be `error` instead of `exception`. 

If we conditionally cal `error` then we can have the function name always be `exception`.
```suggestion
console.exception = function exception() {
  if ("error" in console) {
    console.error.apply(this, arguments);
  }
);
```

Reply:
@JakeChampion Agree

Or it could be even ``console.error.apply(console, arguments)``

My reason why I had not used this way was the functuion name anyway turns into something like ``console.exception = function n() {}`` after minification, so there's no big difference if the name will be ``error`` or ``n``

Reply:
Note that in some old engines, `console` methods are not `Function` instances -> haven't methods like `.apply`.

Reply:
@zloirock Thanks a lot for this critical info

I'll rearrange the code now with ``Function.prototype.call.apply(console.error, arguments)``

Reply:
@zloirock Thanks once again for the remark. Now the polyfill code shouldn't throw even on these rare engines https://github.com/JakeChampion/polyfill-library/pull/1306/commits/bd721604690d51670aed8c7c9e896668df35fdc4